autopush/tests/test_endpoint.py

```diff
@@ -770,10 +770,13 @@ def test_get_valid(self, arg):
 
         def handle_finish(value):
             self.reg.set_status.assert_called_with(200)
-            self.reg.write.assert_called_with(
-                '{"useragentid": "' + dummy_uaid + '", '
-                '"channelid": "' + dummy_chid + '", '
-                '"endpoint": "http://localhost/push/abcd123"}')
+            called_arg = dict(useragentid=dummy_uaid,
+                              channelid=dummy_chid,
+                              endpoint="http://localhost/push/abcd123")
+            call_args = self.reg.write.call_args
+            ok_(call_args is not None)
+            args = call_args[0]
+            eq_(eval(args[0]), called_arg)
```

Comment:
I feel gross inside for using eval, but its the easiest way to turn the str'd dict back into a dict for proper comparison. :disappointed: